persistence/src/main/scala/hmda/persistence/processing/HmdaFileValidator.scala

```diff
@@ -224,17 +208,13 @@ class HmdaFileValidator(submissionId: SubmissionId) extends HmdaPersistentActor
         replyTo ! ValidationCompletedWithErrors(submissionId)
       }
 
-    case VerifyQualityEdits(v) =>
-      persist(QualityEditsVerified(v)) { e =>
-        updateState(e)
-        sender() ! QualityEditsVerified(v)
-      }
-
-    case JustifyMacroEdit(error, j) =>
-      persist(MacroEditJustified(error, j)) { e =>
-        updateState(e)
-        sender() ! MacroEditJustified(error, j)
-      }
+    case VerifyEdits(editType, v) =>
+      if (editType == Quality || editType == Macro) {
```

Comment:
I definitely understand the check for `editType`, but I think this `if` statement might just be clutter.  From my understanding, it's impossible for this statement to fail.